lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -147,6 +149,12 @@ def acceptable_19_syntax_symbol?(sym_name)
             return false
           end
 
+          if cop_config['PreferHashRocketsForQuotedSymbols'] &&
```

Comment:
Please add this configuration key to `config/default.yml` with a default value. We also need to add docs/examples to the cop's top comment for this configuration.